rvm.el

```diff
@@ -344,6 +344,15 @@ function."
 (defun rvm--emacs-gempath ()
   (getenv "GEM_PATH"))
 
+(defun rvm--emacs-bundlepath ()
+  (let ((bundle-config (rvm--call-bundle-process "config get path"))
```

Comment:
The `get` option to `config` looks like it's only supported in Bundler 2.1. In older versions it'll set a config property named `get`. 

I think it makes sense to support older versions. Personally I've yet to work on a project using 2.1.

Reply:
[2.1](https://bundler.io/v2.1/man/bundle-config.1.html) and [2.0](https://bundler.io/v2.0/man/bundle-config.1.html) docs.

Reply:
You’re right. I’ve removed the get command, which is not required.